felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -807,6 +815,12 @@ func (m *endpointManager) resolveWorkloadEndpoints() {
 				m.activeWlIfaceNameToID[workload.Name] = id
 				delete(m.pendingWlEpUpdates, id)
 
+				logCxt.Info("updating QoS bandwidth state if changed")
+				err := m.maybeUpdateQoSBandwidth(oldWorkload, workload)
+				if err != nil {
+					logCxt.WithFields(log.Fields{"oldWorkload": oldWorkload, "newWorkload": workload}).Infof("error updating QoS bandwidth state: %v", err)
```

Comment:
Should an error here be propagated somehow?

Reply:
I'm not sure how, unless we consider adding something to the workloadendpoint's `status`? What do you think?